resources/bisque/guider/connect.js

```diff
@@ -0,0 +1,7 @@
+/* Java Script */
```

Comment:
Given that these files have the `.js` extension, why to they all start with `/* Java Script */`?

Reply:
Because TheSkyX has implemented this in a terrible, terrible fashion (besides the fact that he constantly refers to them as Java scripts).

This is the "special key" that lets TheSkyX know it is valid. 

http://www.bisque.com/scriptTheSkyX/scriptOverSocket.html